tests/integration/istio-injection/istioinjection/fixture.go

```diff
@@ -0,0 +1,161 @@
+package istioinjection
+
+import (
+	"os"
+	"fmt"
+	"math/rand"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	appv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+
+func createNamespace() error {
+	log.Infof("Creating namespace '%s", namespace)
+	_, err := k8sClient.CoreV1().Namespaces().Create(&v1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+			// Labels: map[string]string{
+			// 	"istio-injection": "enabled",
+			// },
+		},
+		Spec: v1.NamespaceSpec{},
+	})
+	if err != nil {
+		log.Errorf("Cannot create namespace '%s': %v", namespace, err)
+		return err
+	}
+	return nil
+}
+
+func deleteNamespace() {
+	log.Infof("Deleting namespace '%s", namespace)
+	var deleteImmediately int64
+	err := k8sClient.CoreV1().Namespaces().Delete(namespace, &metav1.DeleteOptions{
+		GracePeriodSeconds: &deleteImmediately,
+	})
+	if err != nil {
+		log.Errorf("Cannot delete namespace '%s': %v", namespace, err)
```

Comment:
No error handling. Maybe exit with different error code to say "I have run the tests, but haven't cleaned up"